src/puppetlabs/puppetdb/command.clj

```diff
@@ -97,8 +97,8 @@
 
 (def supported-command-versions
   {"replace facts" (version-range 2 4)
-   "replace catalog" (version-range 4 7)
-   "store report" (version-range 3 6)
+   "replace catalog" (version-range 4 8)
```

Comment:
I think that might have maybe been related to the comidi move? I didn't touch anything related to the deprecations.